atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -44,6 +44,28 @@
 
     @Value.Default
     public int poolSize() {
+        return 30;
```

Comment:
Agreed.  Though we could leave the default value to be a derivative of the idle pool size so we don't silently change peoples settings with an upgrade and restart.